workflow_miniscope/paths.py

```diff
@@ -2,8 +2,13 @@
 
 
 def get_miniscope_root_data_dir():
-    return dj.config.get('custom', {}).get('miniscope_root_data_dir', None)
-
+    mini_root_dirs = dj.config.get('custom', {}).get('miniscope_root_data_dir')
+    if not mini_root_dirs:
+        return None
+    elif not isinstance(mini_root_dirs, list):
```

Comment:
use  `collections.abc.Sequence`   instead of `list` to allow other types of sequences.